### PR TITLE
add STM32L433RC, STM32L433RB and STM32L443RC support

### PR DESCRIPTION
**Pull request by emoninet2**

### CODE REVIEW

**Comment by emoninet2** [comment]
I see there are a few Astyle errors. Do I create a new pull request with the errors fixed?

**Comment by fpistm** [comment]
> I see there are a few Astyle errors. Do I create a new pull request with the errors fixed?

Hi @emoninet2.  Thanks for this PR. No, simply  amend your commit with astyle fixes and force push on the same branch. Then this PR will be automatically updated.

File: variants/STM32L4xx/L433R(B-C)(I-T-Y)_L443RC(I-T-Y)/generic_clock.c [hunk 1]
```diff
@@ -24,8 +24,46 @@
   */
 WEAK void SystemClock_Config(void)
 {
-  /* SystemClock_Config can be generated by STM32CubeMX */
-#warning "SystemClock_Config() is empty. Default clock at reset is used."
+  RCC_OscInitTypeDef RCC_OscInitStruct = {};
+  RCC_ClkInitTypeDef RCC_ClkInitStruct = {};
```
Comment (fpistm) [inline variants/STM32L4xx/L433R(B-C)(I-T-Y)_L443RC(I-T-Y)/generic_clock.c:28]:
```suggestion
  RCC_ClkInitTypeDef RCC_ClkInitStruct = {};
  RCC_PeriphCLKInitTypeDef PeriphClkInit = {};
```

**Review by fpistm — CHANGES_REQUESTED** [review]
Thanks for the update.
Still one format issue.
I've also added the USB clock selection else USB will not work as it require 48MHz clock source.

**Review by fpistm — APPROVED** [review]

**Pull request merged**